storage/tables/index.md

```diff
@@ -123,41 +145,79 @@ their uniqueness is checked and the data are de-duplicated. The result table loo
 
 ### Incremental Loading
 When a primary key is defined on a column, it is also possible to take advantage of incremental loads.
 If you load data into a table incrementally, new rows will be added and **existing rows will be updated**.
-No rows will be deleted. If you have a table with a primary key defined on the columns `name` and `age`:
+No rows will be deleted. If you have a table with a primary key defined on the column `name`:
 
-|name|age|money|
-|---|---|---|
-|John|15|$150|
-|John|34|$340|
-|Darla|60|$600|
+|name|money|
+|---|---|
+|John|$150|
+|John|$340|
+|Darla|$600|
 
 and you import the following data to the table:
 
-|name|age|money|
-|---|---|---|
-|Annie|30|$500|
-|John|34|$340000|
-|Darla|60|$600000|
+|name|money|
+|---|---|
+|Annie|$500000|
+|John|$340000|
+|Darla|$600000|
 
 the result table will contain:
 
-|name|age|money|
-|---|---|---|
-|John|15|$150|
-|Darla|60|$600000|
-|John|34|$340000|
-|Annie|30|$500|
+|name|money|
+|---|---|
+|John|$150|
+|Darla|$600000|
+|John|$340000|
+|Annie|$500000|
 
-When importing data into a table with a primary key, the uniqueness is checked. 
-The record `John,34,$340000` will overwrite the row `John,34,$340`, because it has the same primary key.
-The above applies only when **incremental load** is used. 
+When importing data into a table with a primary key, the uniqueness is checked.
+The record `John,$340000` will overwrite the row `John,$340`, because it has the same primary key value.
+The above applies only when **incremental load** is used.
 
-When incremental is not used, the contents of the target table are cleared before the load. When a primary key 
+When incremental is not used, the contents of the target table are cleared before the load. When a primary key
 is not defined and an incremental load is used, it simply appends the data to the table and does not update anything.
 
+### Incremental Processing
+When a table is loaded incrementally the update time of each row is recorded internally. This information
+can be later used in Input Mapping of many components (especially [Transformations](/manipulation/transformations/mappings/#input-mapping)).
+In input mapping, you may specify the `Changed in last` option which allows the component to process only an increment of the data:
+
+{: .image-popup}
+![Screenshot - Incremental Processing](/storage/tables/incremental-processing.png)
+
+Extending the [above example](#incremental-loading) - if you have a table with a primary key defined on the column `name`:
+
+|name|money|
+|---|---|
+|John|$150|
+|Peter|$340|
+|Darla|$600|
+
+and you import the following data to the table:
+
+|name|money|
+|---|---|
+|Annie|$500000|
+|Peter|$340000|
+|Darla|$600000|
+
+assuming that the import was on 2010.01.02 10:00 the result table will contain (the *\*updated\** column is not an actual column of the table, it is just displayed here for illustration purposes):
+
+|name|money|*\*updated\**
+|---|---|---|
+|John|$150|2010.1.1 10:00|

```

Comment:
Každej pes (datum) jiná ves, ani jedno z toho není správně. Hoď to aspoň do ISO :-)

Reply:
2010-01-01T10:00:00Z mi prijde dost necitelny a ve slacku jsme to nedoresili

Reply:
ještě tam bylo pro tyhle případy `2010-01-01 10:00:00`